tools/Google.Cloud.Tools.ReleaseManager/History/HistoryFile.cs

```diff
@@ -112,10 +112,25 @@ internal Section(Release release)
 
                 Lines.Add($"# Version {Version}, released {release.ReleaseDate:yyyy-MM-dd}");
                 Lines.Add("");
+                // TODO(early 2021): Remove this hack (gRPC constructors) once everything has been released with new tooling.
+                bool addGrpcConstructorFaqEntryLink = false;
                 foreach (var commit in release.Commits)
                 {
+                    if (commit.Hash == "0ca05f5")
+                    {
+                        addGrpcConstructorFaqEntryLink = true;
```

Comment:
And maybe don't include the commit's own text?

Reply:
That's what the continue is for :)

Reply:
duh myself